frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -86,9 +86,7 @@ export function Table({
 
   if (typeof clientSidePagination !== 'boolean') clientSidePagination = true;
 
-  const tableTypeProperty = component.definition.styles.tableType;
-  let tableType = tableTypeProperty ? tableTypeProperty.value : 'table-bordered';
-  tableType = tableType === '' ? 'table-bordered' : tableType;
```

Comment:
Removal of logic from here will have effects on backward compatibility.

Reply:
Sure, shifting back to previous logic